tests/templates/kuttl/tls/30-prepare-test-kafka.yaml.j2

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+{% if test_scenario['values']['use-client-auth-tls'] == 'true' %}
+  - script: kubectl cp -n $NAMESPACE ./test_client_auth_tls.sh test-kafka-broker-default-0:/tmp
+  - script: kubectl cp -n $NAMESPACE ./wrong_keystore.p12 test-kafka-broker-default-0:/tmp
```

Comment:
What's the lifetime of the certs in the keystore?

Reply:
yeah good point, generated new ones:
```
openssl x509 -enddate -noout -in ca.crt 
notAfter=Jul 12 09:21:36 2122 GMT
```